src/components/controls/controls.tsx

```diff
@@ -100,7 +102,10 @@ function reducer(state, action) {
 
 export const Controls: React.FC<Props> = ({ onChange }) => {
     const classes = useStyles();
+    initialState.initialNumberOfCases = getCovidData(countryData[0].Country);
+    initialState.totalPopulation = getPopulationData(countryData[0].Country)
```

Comment:
No need to read from countryData. Hard code 'USA' here

Reply:
Done